File: csharp/isogram/Isogram.cs
public static class Isogram
{
    public static bool IsIsogram(string word)
    {
        word = word.ToLower().Replace("-","").Replace(" ","");
        char[] arrWord = word.ToCharArray();
        Array.Sort(arrWord);
        for (int i = 0; i < word.Length - 1; i++)
        {
            if (arrWord[i] == arrWord[i + 1])
            {
                return false;
            }
        }
        return true;
    }
}

File: javascript/isogram/isogram.js
//
// This is only a SKELETON file for the 'Isogram' exercise. It's been provided as a
// convenience to get you started writing code faster.
//

export const isIsogram = (word) => {
  const letters = word.toLowerCase().match(/[a-z]/g) || [];
  return new Set(letters).size === letters.length;
};
